enos/enos-dev-variables.hcl

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
```

Comment:
We should make all license headers be MPL to align with the repo's license. Many of these files are still BUSL. You can do that in a follow-up PR if you prefer.